app/models/katello/docker_tag.rb

```diff
@@ -10,6 +10,8 @@ class DockerTag < Katello::Model
       :complete_value => true, :only_explicit => false
     scoped_search :relation => :docker_manifest, :on => :digest, :rename => :digest,
       :complete_value => false, :only_explicit => true
+    scoped_search :relation => :docker_manifest, :on => :schema_version, :rename => :schema_version,
+      :complete_value => false, :only_explicit => true
```

Comment:
DO you have like an example test  I can base off? I don't want to be testing scoped search.